components/global_header/left_controls/product_menu/product_menu_list/product_menu_list.tsx

```diff
@@ -146,7 +146,7 @@ const ProductMenuList = (props: Props): JSX.Element | null => {
                 <Menu.ItemToggleModalRedux
                     id='userGroups'
                     modalId={ModalIdentifiers.USER_GROUPS}
-                    show={enableCustomUserGroups || isCloud}
+                    show={enableCustomUserGroups || isStarterFree || isStarterFreeTrial}
```

Comment:
It seems that now it will always show and in some conditions it will be disabled? Maybe it was a matter of just moving this condition to disabled like:

```
disabled={!enableCustomUserGroups}
```

And only check for trial for the sibling icon?

Reply:
If we are sure that `enableCustomUserGroups || isStarterFree || isStarterFreeTrial` can be interpreted as "always", then this would be ok.
`enableCustomUserGroups` is calculated checking both:
- FF `CustomGroups` enabled
- Config `EnableCustomGroups` enabled

So, would like to ensure there are no use cases where the item would start to appear when not expected.

Reply:
Ah yes, good catch - I think the FF will go away but probably the settings we still need to preserve